Source/Diffable Collection/FetchedCollectionDiffableDataSource.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  FetchedCollectionDataSource (Diffable).swift
+//  AppwiseCore
+//
+//  Created by Jonathan Provo on 14/03/2022.
+//
+
+import CoreData
+import UIKit
+
+@available(iOS 13.0, *)
+public typealias FetchedDiffableDataSource = UICollectionViewDiffableDataSource<FetchedDiffableSection, FetchedDiffableItem>
+
+@available(iOS 13.0, *)
+public final class FetchedCollectionDiffableDataSource: NSObject, NSFetchedResultsControllerDelegate {
+	// MARK: - Properties
+	/// A boolean indicating whether empty sections should be hidden or not. The default value is `false`.
+	public var isHidingEmptySections: Bool = false
+	/// A boolean indicating whether updated items should be reconfigured or reloaded. Setting this value to `true` will reconfigure items, setting this value to `false` will reload items. The default value is `true`.
+	public var isReconfiguringItems: Bool = true
+	/// A boolean indicating whether differences should be animated. The default value is `true`.
+	public var isAnimatingDifferences: Bool = true
```

Comment:
I'd follow the naming convention of the other sources, i.e. `animateChanges`. Probably apply this to the other bool properties? I'm honestly not sure where we're at with naming conventions for libraries/code/…